spec/features/user_logs_in_spec.rb

```diff
@@ -9,4 +11,20 @@
     assert_equal '/', current_path
     refute page.has_content?("Ryan Asensio")
   end
+
+  it "renders error with bad response", js: true do
+    UserFollowsController.any_instance.stub(:current_user).
+      and_return(User.first)
+    PlacesController.any_instance.stub(:current_place).
+      and_return(Place.first)
+    UsersController.any_instance.stub(:current_user).
+      and_return(User.first)
+    Comment.stub(:for_a_place).
+      and_return(nil)
+    FollowManager.stub(:is_following?).and_return(true)
+    stub_bad_omniauth
+    login_user
+    assert_equal '/', current_path
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.